### PR TITLE
ci: Add workflow for release automation

### PR DESCRIPTION
**Pull request by mayank99**
Added two new workflows:
- `release-please.yml`: uses the [release-please](https://github.com/googleapis/release-please) action to automatically create a release PR and update changelog.
- `pr.yml`: uses a custom script to ensure that PR title follows the conventional commits spec.

Updated:
- `publish.yml`: the release-please action includes the package name in release title, so I'm removing it.
- `CONTRIBUTING.md`: removed details about changelog.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -149,8 +133,7 @@ To enable us to quickly review and accept your pull requests, always create one
   - `.html` file has all possible states of the component.
   - `.js` file has scenarios covering these cases.
   - All existing and new tests should pass.
-- Updated changelog.
 
-After verifying that your changes are ready, you can [create a pull request from your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Make sure that an issue is linked to the pull request and that you have a proper description with screenshots.
+After verifying that your changes are ready, you can [create a pull request from your fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Make sure that the name of your pull request follows the [Conventional Commits spec](https://www.conventionalcommits.org/), and that you have a proper description with screenshots and a [closing keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
```
Comment (bentleyvk) [inline CONTRIBUTING.md:137]: Hm, maybe we need to add this memo to PR template to prevent failing PRs from the start. (at least for me 😃)

Comment (bentleyvk) [reply CONTRIBUTING.md:137]: Oh, this is CSS repo so it doesn't have PR template. Maybe create the same as in React repo?

Comment (mayank99) [reply CONTRIBUTING.md:137]: CSS repo never gets external PRs, so I think we'll be fine here.

**Review by bentleyvk — APPROVED** [review]
Looks good! You will have to demonstrate how to do next release 😃

**Pull request merged**